expat/Makefile.am

```diff
@@ -115,11 +205,88 @@ buildlib:
 	@echo 'ERROR:' >&2
 	@false
 
+#page
+#### building xmlwf
 
-.PHONY: run-benchmark
-run-benchmark:
-	$(MAKE) -C tests/benchmark
-	./run.sh tests/benchmark/benchmark@EXEEXT@ -n $(top_srcdir)/../testdata/largefiles/recset.xml 65535 3
+bin_PROGRAMS = xmlwf/xmlwf
+
+xmlwf_xmlwf_LDADD = libexpat.la
+xmlwf_xmlwf_SOURCES = \
+    xmlwf/xmlwf.c \
+    xmlwf/xmlfile.c \
+    xmlwf/codepage.c \
+    xmlwf/@FILEMAP@.c
+
+xmlwf_xmlwf_CPPFLAGS = -I$(srcdir)/lib
```

Comment:
The `AM_CPPFLAGS = -I$(srcdir)/lib` seems to render this one superfluous.

Reply:
We must  define this variable and  duplicate all the flags  added to `AM_CPPFLAGS` above.   This is because below  we add a flag  to it and defining a  per-executable `_CPPFLAGS` variable causes  `AM_CPPFLAGS` to be ignored by GNU Automake.